Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM_2nd.cpp

```diff
@@ -125,6 +125,11 @@ void FiniteDifferenceSolver::MacroscopicEvolveHMCartesian_2nd(
         Box const &tby = mfi.tilebox(Mfield[1]->ixType().toIntVect());
         Box const &tbz = mfi.tilebox(Mfield[2]->ixType().toIntVect());
 
+        // Extract stencil coefficients for calculating the exchange field H_{ex} and the anisotropy field H_{ani}
+        Real const * const AMREX_RESTRICT coefs_x = m_stencil_coefs_x.dataPtr();
+        Real const * const AMREX_RESTRICT coefs_y = m_stencil_coefs_y.dataPtr();
+        Real const * const AMREX_RESTRICT coefs_z = m_stencil_coefs_z.dataPtr();
```

Comment:
refer to the book in Mendeley: `Qubits for ARTEMIS/Book/Spin Waves Theory and Application` by David Stancil on which parameters are involved in these coefficients. Page 81-91.